lib/experimental/new_embed.dart

```diff
@@ -614,6 +615,109 @@ class FlashBox {
   void hide() {
     _element.setAttr('hidden');
   }
+
+  void show() {
+    _element.clearAttr('hidden');
+  }
+}
+
+class AnalysisResultsElement extends DElement {
+  static const String _noIssuesMsg = 'no issues';
+  static const String _hideMsg = 'hide';
+  static const String _showMsg = 'show';
+
+  DElement message;
+  DElement toggle;
+  bool _hidden = true;
+
+  final StreamController<AnalysisIssue> _onClickSink =
+      StreamController.broadcast();
+  Stream<AnalysisIssue> get onIssueClick => _onClickSink.stream;
+
+  AnalysisResultsElement(Element element, this.message, this.toggle)
+      : super(element) {
+    hide();
```

Comment:
If possible, consider removing this and adding whatever styling is necessary for the "hidden" state to the HTML file. That way we don't risk a FOUC.